docs/references/features-cross-reference.md

```diff
@@ -1231,14 +1231,14 @@
         <td style="text-align: center; color: orange;">●</td>
         <td style="text-align: center; color: orange;">●</td>
         <td style="text-align: center;">N/A</td>
-        <td></td>
+        <td style="text-align: center; color: orange;">● <sup>1</sup></td>
```

Comment:
I am currently thinking about changing to this:

![bildschirmfoto vom 2018-01-02 21-10-00](https://user-images.githubusercontent.com/2174918/34498295-4f55713c-f001-11e7-974a-a8f9d17cf2d3.png)
(The second last column)

What do you think?

Reply:
(Where <code><sup>2</sup></code> is the "only text contents" footnote)

Reply:
@PowerKiKi Any thoughts on this? :)

Reply:
You mean from `●` to `✖` ? I think `●` is the best fit according to legend "Partially supported"

Reply:
Yup, that makes more sense. :)